internal/docker/client.go

```diff
@@ -0,0 +1,230 @@
+package docker
```

Comment:
header license. 
Why it's not under cmd/?

Reply:
Was not sure where to put it and started here. We can move the package, no problem